config/collector.yaml

```diff
@@ -76,10 +133,24 @@ processors:
         new_label: version
         new_value: Google-Cloud-OTLP manifests:0.2.0 OpenTelemetry Collector Built By Google/0.121.0 (linux/amd64)
 
+  # The resourcedetection processor is configured to detect GCP resources.
+  # Resource attributes that represent the GCP resource the collector is
+  # running on will be attached to all telemetry that goes through this
+  # processor.
```

Comment:
Resource detection is required, otherwise you will write time series from many resources into the same time series in GCP and will get a lot of errors.

Reply:
Addressed in [27ae370](https://github.com/GoogleCloudPlatform/otlp-k8s-ingest/pull/52/commits/27ae370d95a5685d655fa3b2415e8a433ca642eb)